tests_netcore/Plivo.NetCore.Test/Resources/TestMessage.cs

```diff
@@ -39,7 +39,25 @@ public void TestMessageCreate()
                     response,
                     Api.Message.Create(src:"+919999999999", dst:new List<string>() {"+919898989898", "+919090909090"},
                        text:"textext")));
+
             AssertRequest(request);
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(src: "+919999999999", dst:  "+919898989898<+919090909090",
+                      text: "textext")));
+
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(dst: "+919898989898<+919090909090",
+                      text: "textext",powerpack_uuid:"asdasd-asdasd-asdasd21-asd")));
+
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(dst: new List<string>() { "+919898989898", "+919090909090" },
```

Comment:
As per your change, dst is a string param, but  here your passing list<string> as dst param. is this working fine?

Reply:
Yes, that's the idea to support both String and List.